code/workspaces/web-app/src/components/settings/UserPermissionsTable.js

```diff
@@ -166,6 +166,7 @@ export function UserPermissionsTableBody({ users, currentUserId, project, classe
       setRemoveUserDialogState={setRemoveUserDialogState}
       actions={actions}
       dispatch={dispatch}
+      key={`row-${index}`}
```

Comment:
Wouldn't it be better to have
`key={user.userid}`
(index being a last resort, see https://reactjs.org/docs/lists-and-keys.html#keys)

Reply:
Good idea, thank you for pointing me to that document! I will update to use the ID.